lib/v2/iiilab/index.js

```diff
@@ -0,0 +1,24 @@
+const buildData = require('@/utils/common-config');
+const baseUrl = 'https://www.iiilab.com/';
+
+module.exports = async (ctx) => {
+    const link = String(baseUrl);
```

Comment:
```suggestion
    const link = baseUrl;
```
baseUrl is a string, no need to type cast it again